dashboard/app/controllers/test_controller.rb

```diff
@@ -175,4 +190,127 @@ def destroy_level
     level.destroy
     head :ok
   end
+
+  # Use the same data from pd_teacher_application_hash_common
+  # We can't use the factory directly because FactoryGirl is not available on prod
+  def teacher_form_data
+    {
+      'school': School.first.id,
+      'country': 'United States',
+      'first_name': 'First',
+      'last_name': 'Last',
+      'alternate_email': 'ilovepotions@gmail.com',
+      'phone': '5558675309',
+      'gender_identity': 'Male',
+      'race': ['Other'],
+      'street_address': '333 Hogwarts Place',
+      'city': 'Magic City',
+      'state': 'Washington',
+      'zip_code': '98101',
+      'principal_role': 'Headmaster',
+      'principal_first_name': 'Albus',
+      'principal_last_name': 'Dumbledore',
+      'principal_title': 'Dr.',
+      'principal_email': 'socks@hogwarts.edu',
+      'principal_confirm_email': 'socks@hogwarts.edu',
```

Comment:
I am SO amused by these emails 😂

Reply:
It wasn't me! This was a copy and paste effort 😬